maquetado/scss/login.scss

```diff
@@ -0,0 +1,78 @@
+.login-container {
+  background-color: $wild-sand;
+  display: flex;
+  flex-flow: column wrap;
+  height: 100%;
+  justify-content: center;
+  max-height: 400px;
+  max-width: 300px;
+  width: 100%;
+
+  .login-top {
+    .top-decorator {
+      background-color: $cerulean;
+      border: 2px solid $cerulean;
```

Comment:
If the background color is the same that the border, so the border is not necessary

Reply:
But, if i remove the border all color disappear! :eyes: